Caesar_cipher/gonvers/main.py

```diff
@@ -0,0 +1,45 @@
+class CaesarCypher():
+    chars = "ABCDEFGHIJKLMNOPQRSTUVWXYZ"
+    chars_len = len(chars)
+    def __init__(self, key=0):
+        self.key = key
+    def set_key(self, key):
```

Comment:
You do not need set_key and get_key methods. You can change it by `cypher.key = 4` for example.